syntax = "proto3";

message Person {
  int32 id = 1;
  string first_name = 2;
  string last_name = 3;
  string company_name = 4;
}

message PersonList {
  repeated Person persons = 1;
}

message PersonQuery {
  int32 id = 1;
}

service PersonService {
    rpc Filter(PersonQuery) returns (PersonList);
}